src/main/java/de/bonndan/nivio/output/map/svg/SVGBackgroundFactory.java

```diff
@@ -9,6 +9,11 @@
 
 import static java.lang.Math.round;
 
+/**
+ * This generates the background hex tiles.
+ *
+ * TODO It is a bad bank for technical debt.
```

Comment:
What technical debt do you think that we are collecting here? For me it is just that I don't really understand what is going on (but I also haven't fully understood the other rendering logics).

Reply:
Here we produce hundreds of hexagons to display them in the background. There is no real added value and no business logic, but the designer suggested it like that.

Reply:
I have refactored it a bit and added some comments.

Reply:
Nice, it is better readble now!